lib/node_modules/@stdlib/array/empty-like/docs/types/index.d.ts

```diff
@@ -0,0 +1,618 @@
+*
+* @param x - input array from which to derive the output array length
+* @param dtype - data type
+* @returns empty array
+*
+* @example
+* var zeros = require( `@stdlib/array/zeros` );
+*
+* var x = zeros( 2, 'float32' );
+* // returns <Float32Array>[ 0.0, 0.0 ]
+*
+* var arr = emptyLike( x, 'uint8' );
+* // returns <Uint8Array>
+*/
+declare function emptyLike( x: Uint8Array ): Uint8Array;
+
+/**
+* Creates an uninitialized array having the same length as a provided input array.
+*
+* ## Notes
+*
+* -   In browser environments, the function always returns zero-filled arrays.
+* -   In Node.js versions `>=3.0.0`, the underlying memory of returned typed arrays is **not** initialized. Memory contents are unknown and may contain **sensitive** data.
+*
+* @param x - input array from which to derive the output array length
+* @param dtype - data type
+* @returns empty array
+*
+* @example
+* var zeros = require( `@stdlib/array/zeros` );
+*
+* var x = zeros( 2, 'float32' );
+* // returns <Float32Array>[ 0.0, 0.0 ]
+*
+* var arr = emptyLike( x, 'uint8c' );
+* // returns <Uint8ClampedArray>
+*/
+declare function emptyLike( x: Uint8ClampedArray ): Uint8ClampedArray;
+
+/**
+* Creates a zero-filled array having a specified length.
+*
+* @param x - input array from which to derive the output array length
+* @param dtype - data type
+* @returns zero-filled array
+*
+* @example
+* var zeros = require( `@stdlib/array/zeros` );
+*
+* var x = zeros( 2, 'float32' );
+* // returns <Float32Array>[ 0.0, 0.0 ]
+*
+* var arr = emptyLike( x, 'generic' );
+* // returns
+*/
+declare function emptyLike( x: Array<any> ): Array<any>;
+
+/**
+* Creates an uninitialized array having the same length as a provided input array.
+*
+* ## Notes
+*
+* -   In browser environments, the function always returns zero-filled arrays.
+* -   If `dtype` is `'generic'`, the function always returns a zero-filled array.
+* -   In Node.js versions `>=3.0.0`, the underlying memory of returned typed arrays is **not** initialized. Memory contents are unknown and may contain **sensitive** data.
+*
+* The function recognizes the following data types:
+*
+* -   `float64`: double-precision floating-point numbers (IEEE 754)
+* -   `float32`: single-precision floating-point numbers (IEEE 754)
+* -   `complex128`: double-precision complex floating-point numbers
+* -   `complex64`: single-precision complex floating-point numbers
+* -   `int32`: 32-bit two's complement signed integers
+* -   `uint32`: 32-bit unsigned integers
+* -   `int16`: 16-bit two's complement signed integers
+* -   `uint16`: 16-bit unsigned integers
+* -   `int8`: 8-bit two's complement signed integers
+* -   `uint8`: 8-bit unsigned integers
+* -   `uint8c`: 8-bit unsigned integers clamped to `0-255`
+* -   `generic`: generic JavaScript values
+*
+* @param x - input array from which to derive the output array length
+* @param dtype - data type (default: 'float64')
```

Comment:
```suggestion
* @param dtype - data type
```

There is no default dtype. It is inferred from the input array.